### PR TITLE
Added coloring to tr on main page

### PR DESCRIPTION
**Pull request by Appleguy15**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -385,8 +385,23 @@ function subval_sort($a, $subkey, $result) {
...
 				}
 				}
-				?>
+                //Query for globals
+        $globals_query = "SELECT starttime FROM globals";
+        $globals_result = $db_server->query($globals_query);
+        $globals_data = $globals_result->fetch_array();
+        $todaytime = new DateTime();
+        $todaytimestart = new DateTime($globals_data['starttime']);
+
```
Comment (nwarmenh) [inline index.php]: What's with your indentation here?  Clean it up?

**Comment by Appleguy15** [comment]
@nwarmenh I cleaned up the code.

**Comment by badtzmaru** [comment]
Maybe you did, but you sure didn't commit it.

**Comment by badtzmaru** [comment]
There's only one commit still @Appleguy15

**Comment by Appleguy15** [comment]
I have been having weird commit error. I will try it again when I get to
school. @badtzmaru

**Comment by badtzmaru** [comment]
Alright

**Comment by Appleguy15** [comment]
@badtzmaru Just recommited

**Comment by Appleguy15** [comment]
@badtzmaru It is not working. When I ran it on local host all the names are red and it is before the starttime.

**Pull request closed without merging**